t4c21/t4c_lightning_datamodule.py

```diff
@@ -183,12 +275,17 @@ def __init__(
         self.num_workers = num_workers
 
         self.dataset_cls = dataset_cls
+        self.test_dataset_cls = test_dataset_cls
         self.dataset_parameters = dataset_parameters
         self.dataloader_config = dataloader_config
+        self.test_dataset_parameters = test_dataset_parameters
 
     @overrides
     def setup(self, *args, **kwargs) -> None:
         self.dataset = self.dataset_cls(**self.dataset_parameters)
+        self.test_dataset = None
```

Comment:
I would prefer oneliners :) like: 
< 
self.test_dataset = self.test_dataset_cls(**self.test_dataset_parameters) if self.test_dataset_parameters is not None else None

Reply:
@chrisworld matter of taste as you say. I'm not a huge fan of the "positive value" "condition" "negative value" order in python as it goes against the evaluation flow. On the other hand, I extensively use list comprehensions, but it can get non-readable, too.